### PR TITLE
feat: add renku wallpaper and set it as the default

### PR DESCRIPTION
**Pull request by gavin-k-lee**
adds a renku branded wallpaper to the vnc

### CODE REVIEW

File: docker/vnc/Dockerfile [hunk 1]
```diff
@@ -58,6 +58,13 @@ COPY gitk.desktop /home/jovyan/Desktop/
 COPY .git_icon.png /home/jovyan/Desktop/
 RUN chmod +x /home/jovyan/Desktop/gitk.desktop
 
+#################################################################
+# Add wallpaper and force the default to point to the renku one
+
+RUN mkdir -p /usr/share/backgrouns/renku/
+COPY renku_background_dots.png /usr/share/backgrounds/renku/
+RUN ln -s -f /usr/share/backgrounds/renku/renku_background_dots.png /usr/share/backgrounds/xfce/xfce-stripes.png
```
Comment (rokroskar) [inline docker/vnc/Dockerfile:66]: that's dirty! :)

File: docker/vnc/Dockerfile [hunk 2]
```diff
@@ -58,6 +58,13 @@ COPY gitk.desktop /home/jovyan/Desktop/
 COPY .git_icon.png /home/jovyan/Desktop/
 RUN chmod +x /home/jovyan/Desktop/gitk.desktop
 
+#################################################################
+# Add wallpaper and force the default to point to the renku one
+
+RUN mkdir -p /usr/share/backgrouns/renku/
```
Comment (rokroskar) [inline docker/vnc/Dockerfile:64]:
```suggestion
RUN mkdir -p /usr/share/backgrounds/renku/
```

**Review by rokroskar — CHANGES_REQUESTED** [review]

File: docker/vnc/Dockerfile (hunk 1 above)
Comment (gavin-k-lee) [reply docker/vnc/Dockerfile:66]: Yeah I know... 

The thing is the local `~/.config/xfce4/xfconf/xfce-perchannel-xml/xfce4-desktop.xml` profile (where we can replace the stripes.png with the renku.png) isn't generated until the VNC button is clicked.

The alternative is to do it 'cleanly' in the `post-init.sh` script, where we spin up the VNC and then in that `xfce4-desktop.xml` file do a `sed` to replace it properly. 

What do you think?

Comment (rokroskar) [reply docker/vnc/Dockerfile:66]: I think this is fine - it just might be hard to find the image in case someone goes looking for it

Comment (gavin-k-lee) [reply docker/vnc/Dockerfile:66]: True. If I try to go into the desktop settings from the desktop, I get shown 

image

so there's hopefully not really any user error, unless as you say they go looking for that specific wallpaper.

**Review by rokroskar — APPROVED** [review]
Thanks @gavin-k-lee this is starting to look great!

**Comment by rokroskar** [comment]
btw you should update the screenshot in the discourse post ;)

**Comment by gavin-k-lee** [comment]
Yep that was the next thing on my to-dos

**Review by rokroskar — APPROVED** [review]

**Pull request merged**